Makefile

```diff
@@ -358,14 +358,31 @@ androidupdate: all
 	# assets are compressed manually and stored inside the APK.
 	cd $(INSTALL_DIR)/koreader && 7z a -l -mx=9 -mfb=256 -mmt=on \
 		../../$(ANDROID_LAUNCHER_DIR)/assets/module/koreader-$(VERSION).zip * \
+		-xr!*cache$ \
+		-xr!*clipboard$ \
+		-xr!*data/dict$ \
+		-xr!*data/tessdata$ \
+		-xr!*history$ \
+		-xr!*l10n/.tx$ \
+		-xr!*l10n/templates$ \
+		-xr!*ota$ \
 		-xr!*resources/fonts* \
 		-xr!*resources/icons/src* \
+		-xr!*rocks/bin$ \
+		-xr!*rocks/lib/luarocks$ \
+		-xr!*screenshots$ \
 		-xr!*share/man* \
 		-xr!*spec$ \
+		-xr!*tools$ \
 		-xr!*COPYING$ \
+		-xr!*LICENSE$ \
```

Comment:
I probably overlooked that before, but is that a good idea to skip?

Reply:
There are a bunch of LICENSE files, all buried in /data/data/org.koreader/launcher/files, which are not available to end users.

Probably a generic license, like https://github.com/koreader/koreader/blob/master/platform/debian/copyright, on top of the APK is more useful than something buried into assets and extracted to private app dirs.

Reply:
All LICENSE files re-included for now. It is a major PITA to aknowledge all licenses in single document.